docs/interactive-pause.md

```diff
@@ -40,10 +30,31 @@ vendor/bin/mftf run:manifest
 vendor/bin/mftf run:failed
 ```
 
-### MFTF Codecept Run Command
+### Use `Interactive Pause` During Test Development
+
+Here is a typical work flow for this use case:
+ 
+- Set `ENABLE_PAUSE` to `true` under `.env`
+- Add `<pause>` action in a test where you want to stop for investigation
+- Run test
+- Execution should pause at <pause> action and invoke interactive console
+- Try out commands in interactive console
+- Resume test execution by pressing `ENTER`
+
+### Use `Pause` On Test Failure
+
+When `ENABLE_PAUSE` is set to `true`, MFTF automatically generates `pause()` action in `_failed()` hook for tests and in `_failed()` function in `MagentoWebDriver`.
+This allows you to use `pause` to debug test failure for a long running test. The work flow might look like:
+
+- Set `ENABLE_PAUSE` to `true` under `.env`
+- Run test
+- Execution pauses and invokes interactive console right after test fails
+- Examine and debug on the spot of failure
+
+## MFTF Codecept Run Command
 
 You can also use MFTF's wrapper command to run Codeception directly and activate `Interactive Pause` by passing `--debug` option. 
-You do not need to set `ENABLE_PAUSE=true` for this command.
+You do not need to set `ENABLE_PAUSE=true` for this command if you are not using `Pause on Failure`.
```

Comment:
You do not need to set `ENABLE_PAUSE=true` for this command if you don't want to pause on test failure.

Reply:
Replaced